ci/run_rubocop_specs

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
should this file have `set -eux`, so commands aren't allowed to fail?

Reply:
Thanks, fixed